contracts/coordinator/src/contract.rs

```diff
@@ -52,6 +52,15 @@ pub fn query(deps: Deps, _env: Env, msg: QueryMsg) -> Result<Binary, ContractErr
         QueryMsg::GetActiveVerifiers { chain_name } => {
             to_binary(&query::get_active_worker_set(deps, chain_name)?).map_err(|err| err.into())
         }
+        QueryMsg::CheckWorkerCanUnbond {
```

Comment:
```suggestion
        QueryMsg::ReadyToUnbond {
```
I find "can unbond" a confusing term because it sounds like we're checking if the worker is allowed to unbond. From how the response is used that is not the case: If the result is `false`, the service registry starts the unbonding process for the worker, if it's `true` it completes it. So I advocate for a rename

Reply:
Makes sense. Thanks.